aws/src/main/java/org/apache/iceberg/aws/AwsProperties.java

```diff
@@ -735,4 +788,73 @@ private Set<Tag> toTags(Map<String, String> properties, String prefix) {
   public Map<String, String> s3BucketToAccessPointMapping() {
     return s3BucketToAccessPointMapping;
   }
+
+  public boolean s3KeyIdAccessKeyConfigured() {
```

Comment:
These 2 can be private methods now

Reply:
I made both private and renamed `s3KeyIdAccessKeyConfigured` to `s3KeyIdAccessKeyBothConfigured` to reveal its purpose more clearly. For `credentialsProvider`, I also delete its test case added in my previous commit since it is now a private method.